apps/nuxt3/nuxt.config.ts

```diff
@@ -1,11 +1,4 @@
 import { defineNuxtConfig } from 'nuxt/config'
 
 // https://v3.nuxtjs.org/api/configuration/nuxt.config
-export default defineNuxtConfig({
-  plugins: [
-    {
-      src: '~/plugins/vue3-particles.ts',
-      mode: 'client',
-    },
-  ],
-})
+export default defineNuxtConfig({})
```

Comment:
Why this was removed?

Reply:
Nuxt automatically imports any plugin in the `plugins` directory; this section was redundant.

Reply:
Oh ok, didn’t know that

Reply:
Oh looking at the plugin, there's the `mode: 'client'`, to avoid SSR, I don't think it does that automatically.

Reply:
Whoops, good catch! Nuxt uses the `.client.ts` suffix for that now instead of having you write a declaration.